vars/transportRequestRelease.groovy

```diff
@@ -28,14 +36,43 @@ def call(parameters = [:]) {
 
         def script = parameters?.script ?: [commonPipelineEnvironment: commonPipelineEnvironment]
 
-        ChangeManagement cm = new ChangeManagement(script)
+        ChangeManagement cm = parameters.cmUtils ?: new ChangeManagement(script)
 
         Map configuration = ConfigurationMerger.merge(script, STEP_NAME,
                                                       parameters, parameterKeys,
                                                       stepConfigurationKeys)
 
         def changeDocumentId = configuration.changeDocumentId
-        if(!changeDocumentId) throw new AbortException("Change document id not provided (parameter: 'changeDocumentId').")
+
+        if(changeDocumentId?.trim()) {
+
+            echo "[INFO] ChangeDocumentId '${changeDocumentId}' retrieved from parameters."
+
+        } else {
+
+            echo "[INFO] Retrieving ChangeDocumentId from commit history [from: ${configuration.gitFrom}, to: ${configuration.gitTo}]." +
+                 "Searching for pattern '${configuration.gitChangeDocumentLabel}'. Searching with format '${configuration.gitFormat}'."
+
+            try {
+                changeDocumentId = cm.getChangeDocumentId(
+                                                  configuration.gitFrom,
+                                                  configuration.gitTo,
+                                                  configuration.gitChangeDocumentLabel,
+                                                  configuration.gitFormat
+                                                 )
+
+                echo "[INFO] ChangeDocumentId '${changeDocumentId}' retrieved from commit history"
+
+            } catch(ChangeManagementException ex) {
+                echo "[WARN] Cannot retrieve changeDocumentId from commit history: ${ex.getMessage()}."
+            }
+        }
+
+        if(!changeDocumentId) {
+            throw new AbortException("Change document id not provided (parameter: 'changeDocumentId' or via commit history).")
```

Comment:
Error does exactly the same: it thows an AbortException. See: https://github.com/jenkinsci/workflow-basic-steps-plugin/blob/7955d583910839d8eee469e92558567d6a46121d/src/main/java/org/jenkinsci/plugins/workflow/steps/ErrorStep.java#L62

I suggest not to change this since this will be changed anyhow when we use the new ConfigHelper features here.